crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -20,3 +20,194 @@ async fn create_job_fails_job_already_exists() {
 async fn create_job_fails_works_new_job() {
     // TODO
 }
+
+#[cfg(test)]
+mod job_handler_tests {
+    use std::collections::HashMap;
+    use std::sync::Arc;
+
+    use mockall::predicate::eq;
+    use rstest::rstest;
+    use uuid::Uuid;
+
+    use crate::config::config;
+    use crate::jobs::constants::{JOB_PROCESS_ATTEMPT_METADATA_KEY, JOB_VERIFICATION_ATTEMPT_METADATA_KEY};
+    use crate::jobs::job_handler_factory::mock_factory;
+    use crate::jobs::types::{ExternalId, JobItem, JobStatus, JobType, JobVerificationStatus};
+    use crate::jobs::{create_job, process_job, verify_job, Job, MockJob};
+    use crate::tests::common::drop_database;
+    use crate::tests::config::TestConfigBuilder;
+
+    #[rstest]
+    #[case(JobType::SnosRun, false, true)]
+    #[case(JobType::ProofCreation, true, true)]
+    #[should_panic]
```

Comment:
we should add the error message for the panic. eventually we will move this to an error enum after Heemank does the PR